extensions/intellij/intellij_generator_plugin/build.gradle

```diff
@@ -25,7 +25,7 @@ compileTestKotlin {
     kotlinOptions.jvmTarget = "1.8"
 }
 intellij {
-    version '2018.2.7'
+    version '2019.1.0'
```

Comment:
This should be `2019.1`. I'll update it and bump up the plugin version, and do a new release ;)

Reply:
Actually, this does not seem to be needed...

Reply:
<img width="904" alt="Screen Shot 2019-04-05 at 7 36 20 AM" src="https://user-images.githubusercontent.com/3237451/55628146-88d59680-5775-11e9-8027-74a3a7242f01.png">

It seems it is defaulted to the latest version, so... I will remove it ;)